source/TrailMapper/TrailMapper/AppDelegate.swift

```diff
@@ -32,6 +32,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             print("could not start reachability notifier")
         }
 
+        // To check that is there any recording is going or not
+        TMUtility.sharedInstance.checkForRecodringStatus()
```

Comment:
Does this work? There is a typo Recodring -> Recording

Reply:
Thanks, i will correct this typo mistake.